3.10/features-community-edition.md

```diff
@@ -0,0 +1,191 @@
+---
+layout: default
+---
+# Community Edition Features
+
+<https://www.arangodb.com/community-server/>
+
+## General
+
+- **Graph Database**:
+  You can model complex domains because edges are documents without any
+  restrictions in complexity.
+
+- **Document Database**:
+
+- **Schema-free**:
```

Comment:
general note - please add missing descriptions and links to this topic as well

Reply:
Still not sure what master/master conflict resolution means (got no response from engineering). Some of the points could use more polishing (see HTML comments). Unclear if EE descriptions should be revised to include less technical details.